code/modules/virus2/disease2.dm

```diff
@@ -1,4 +1,5 @@
 var/global/list/disease2_list = list()
+var/global/list/virusDB = list()
```

Comment:
That line was there before, I simply moved it from the middle of the file to the top.

I think it's used in the medical laptop and HUD.